routes/laravel.php

```diff
@@ -16,6 +16,6 @@
         })->name('scribe.postman');
 
         Route::get("$prefix.openapi", function () {
-            return response()->file(Storage::disk('local')->path('scribe/openapi.yaml'));
+            return response()->file(Storage::disk('local')->path('scribe/openapi.' . (config('scribe.openapi.json', false) ? 'json' : 'yaml')));
```

Comment:
I feel that this will be more unified and elegant `OpenAPISpecWriter::getSpecFileName()`